legged_control_node:
  ros__parameters:
    control_frequency: 50.0 # 控制频率 (Hz)

    policy_path: "policy.pt"

    # leg_joint2motor_idx: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
    leg_joint2motor_idx: [3, 4, 5, 0, 1, 2, 9, 10, 11, 6, 7, 8]

    # kps: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
    # kds: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
    kps: [40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0]
    kds: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
    # kps: [40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0, 40.0]
    # kds: [3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0]
    # kps: [60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0, 60.0]
    # kds: [5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0]
    
    # default_angles_stand: [-0.1,  0.8, -1.5,
    #                         0.1,  0.8, -1.5,
    #                        -0.1,  1.0, -1.5,
    #                         0.1,  1.0, -1.5 ]
    default_angles_stand: [ 0.1,  0.8, -1.5,
                           -0.1,  0.8, -1.5,
                            0.1,  1.0, -1.5,
                           -0.1,  1.0, -1.5 ]
    
    # default_angles_lie: [-0.25,  1.15, -2.70,
    #                       0.25,  1.15, -2.70,
    #                      -0.50,  1.20, -2.70,
    #                       0.50,  1.20, -2.70 ]
    default_angles_lie: [ 0.25,  1.15, -2.70,
                         -0.25,  1.15, -2.70,
                          0.50,  1.20, -2.70,
                         -0.50,  1.20, -2.70 ]

    num_actions: 12
    num_obs: 45

    ang_vel_scale: 0.25
    dof_pos_scale: 1.0
    dof_vel_scale: 0.05
    action_scale: 0.25

    cmd_scale: [2.0, 2.0, 0.25]
    max_cmd: [0.8, 0.5, 1.57]
